app/src/main/res/layout/item_simpleitem.xml

```diff
@@ -26,14 +26,18 @@
                 android:layout_marginStart="@dimen/spacing_large"
                 android:layout_marginEnd="0dp"
                 android:textStyle="bold"
-                android:text="@string/name"/>
+                android:text="@string/name"
+                android:layout_marginLeft="@dimen/spacing_large"
+                android:layout_marginRight="0dp" />
```

Comment:
Not needed

Reply:
its needed because of the minSdkLevel. Start and End are only supported in later versions

Reply:
In that case can you move them under start and end?